.readthedocs.yml

```
@@ -5,6 +5,12 @@
 # Required
 version: 2
 
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.8"
```

```suggestion
    python: "3.10"
```

Let's try 3.10 per @novafacing 's recommendation.